### PR TITLE
CW+HW 20221118

### PR DESCRIPTION
**Pull request by awesomeprogrammer1**
# what

CW+HW 20221118

# why

Saving work

### CODE REVIEW

File: code_school/files/work_files/email-assignments/email4_assignment2.py [hunk 1]
```diff
@@ -12,37 +12,46 @@
...
-        file = open(file_path, "r")
-        artist_db = json.load(file)
-        file.close()
         artist_name = enterbox(
             "Please enter the name of a musical artist", "Add Artist"
         )
         # artist_db = {"Andrew": {"Help":100, "Me": 132567}}
         # add_artist = "Andrew"
-        if artist_name in artist_db.keys():
+        if artist_name in load_db():
```
Comment (sryabkov) [inline code_school/files/work_files/email-assignments/email4_assignment2.py:46]: Please explain what happens when we call `load_db()` and what happens with the results of this operation

File: code_school/lists/sets_presentation.ipynb [hunk 2]
```diff
@@ -35,7 +35,27 @@
...
+   "source": [
+    "vegetables = set((\"broccoli\",\"zucchini\",\"spinach\"))\n",
+    "print(vegetables)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Sets are changeable*, however they are not ordered via indexes\n",
```
Comment (sryabkov) [inline code_school/lists/sets_presentation.ipynb:55]: How do you say "changeable" and "not changeable" in computer speak?

**Review by sryabkov — COMMENTED** [review]
#### Sets

Why is 

> Sets, similar to lists, are used to store multiple items in a single variable

explained in one place and 

```
A Set can't hold duplicate items
```
is in the comment?

How do you find elements in one set that are not in the other set? Difference, but not symmetric

How do you check if something is in a set or not?

What happens if you try to add an element to the set that is already in the set?

#### `code_school/lists/if_str_and_list.ipynb`

Similar to the above, please decide where you are explaining things (in the text or in the comments) and stick to one place.


> While with lists, it will only check if the words themselves are in the elements

Bad English, please rephrase and provide a better explanation

> This is because in will check if the thing we are searching for is part of a sequence.

Bad explanation. And you are using "sequence". What is "sequence" in this context?

---

See inline comments

File: code_school/lists/sets_presentation.ipynb (hunk 2 above)
Comment (awesomeprogrammer1) [reply code_school/lists/sets_presentation.ipynb:55]: Mutable and immutable

File: code_school/files/work_files/email-assignments/email4_assignment2.py (hunk 1 above)
Comment (awesomeprogrammer1) [reply code_school/files/work_files/email-assignments/email4_assignment2.py:46]: Load DB is essentially just artist_db except we have it in advance, already in memory ready to be called into operation

**Pull request merged**